Documentation/Playground.md

```diff
@@ -0,0 +1,143 @@
+# The Playground App
+
+The Playground app, which is the only in-repo Babylon Native app supported 
+on all platforms, evolved from a simple test harness designed to run 
+small bespoke JavaScript scripts in order to exercise under-development
+code paths. This is still the most common usage of the Playground, and 
+will likely remain so for many developers. However, the long-term vision
+of the Playground app is significantly more sophisticated and may open the
+door for new use cases, both for the Playground app and for Babylon Native 
+in general.
+
+## The Current State
+
+At present, the Playground app is primarily used to execute 
+[experience.js](../Apps/Playground/Scripts/experience.js); most commonly, 
+developers will modify experience.js to test whatever feature is currently
+being developed, sometimes checking in the change with an enabling flag if
+they feel that other developers may also benefit from the same test. 
+There are, however, a few additional features and use cases that are less 
+obvious.
+
+Though designed to execute experience.js by default, the Playground app is 
+also capable of loading and executing arbitrary JavaScript files. How this 
+is done varies by platform -- Win32 allows JS files to be passed as 
+command line arguments while UWP allows them to be executed via protocal 
+invocation, for example -- but the behavior is the same on all platforms
+where invocation is supported: the app will first load the Babylon.js 
+scripts (Babylon itself, the glTF loaders, etc.), then execute the 
+requested external JS file, then execute a special "playground runner" 
+intended to call behaviors that may have been defined in the requested JS
+file. Essentially, this allows for externally-defined JS files to control 
+the behavior of the Playground app as an alternative to experience.js --
+at runtime and without the need to recompile or redeploy.
+
+The Playground app also features a "hot reload" capability on certain
+platforms (Win32 and UWP, for now). The current implementation is that 
+pressing a button (`R` at present, likely to be remapped to `F5` in the 
+future) while the app is in focus causes the Playground to tear down 
```

Comment:
> (`R` at present, likely to be remapped to `F5` in the future)

I recommend removing this and things like this to avoid having to update this documentation when things change.

Reply:
Unless you feel strongly about this one, I'd prefer to leave this and have us take on the responsibility of keeping these docs up-to-date because I feel we need a place we can send new Babylon Native developers and say, "Read this," and have it really tell them the sorts of things they can do. This documentation page in particular will, I suspect, need major updates in the near future anyway as the vision/implementation for the Playground app develops. It is a bit awkward to have documentation specifically calling out things that are a certain way now but will likely change in known ways soon. The whole `Playground.md` is like that, though, and my intention for it was to serve as a place we could direct the people who are looking to jump into Babylon Native right now so that they could get context on the current conversation about the Playground app (incomplete as that conversation is) and, if they want to, get involved. Thoughts?

Reply:
Either way is fine.